AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Support/MutationEvent+Extensions.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Dispatch
+import AWSPluginsCore
+
+extension MutationEvent {
+
+    // updates the head of pending mutation event queue for a given model `id`
+    // if it has a `nil` version, with syncMetadata version in `mutationSync`
+    // and saves it in the mutation event table
+    static func updatePendingMutationEventVersionIfNil(for modelId: Model.Identifier,
+                                                       mutationSync: MutationSync<AnyModel>,
+                                                       storageAdapter: StorageEngineAdapter,
+                                                       completion: @escaping DataStoreCallback<Void>) {
+        MutationEvent.pendingMutationEvents(
+            for: modelId,
+            storageAdapter: storageAdapter) { queryResult in
+            switch queryResult {
+            case .failure(let dataStoreError):
+                completion(.failure(dataStoreError))
+            case .success(let localMutationEvents):
+                guard var existingEvent = localMutationEvents.first else {
+                    Amplify.log.verbose(
+                        "\(#function) Mutation event table doesn't have mutation events with model id : \(modelId)")
```

Comment:
if the mutation event table doesn't have a mutation event with the id, that just means that there are no more pending mutations and nothing to sync the response version with. if this is called for every sync to cloud mutation, we'll get this log line every time. because this is common and would result in a lot of log lines if verbose is turned on, i'm not sure what the information from this log line is providing. can we just remove this log line completely?